cpp/include/cuml/fil/fil.h

```diff
@@ -240,6 +243,9 @@ struct treelite_params_t {
   float threshold;
   // storage_type indicates whether the forest should be imported as dense or sparse
   storage_type_t storage_type;
+  // blocks_per_sm, if set, works as a limit to improve cache hit rate for larger forests
```

Comment:
It would be better to explain what special values are there for `blocks_per_sm`. E.g. "if 0, launch ceildiv(num_rows, NITEMS) blocks".